### PR TITLE
Merge master 5.0.1xx preview8

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

File: src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets [hunk 1]
```diff
@@ -19,14 +19,6 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Import Project="Microsoft.NET.WindowsSupportedTargetPlatforms.props" />
   </ImportGroup>
 
-  <!--
-    Workaround: https://github.com/microsoft/msbuild/issues/4948
-    Allow opt-out of importing framework's winfx.targets.
-  -->
-  <PropertyGroup>
-    <ImportFrameworkWinFXTargets Condition="'$(ImportFrameworkWinFXTargets)' == ''">true</ImportFrameworkWinFXTargets>
```
Comment (wli3) [inline src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets:27]: @BenVillalobos this merge looks good to you?

Comment (BenVillalobos) [reply src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets:27]: Looks good 👍

**Review by benvillalobos — APPROVED** [review]
The `ImportFrameworkWinFXTargets` change LGTM.

**Comment by wli3** [comment]
Weird why this is still failing

**Pull request merged**